### PR TITLE
Dashboard page, Product Info Page and E2E cases for both the pages

### PR DESCRIPTION
**Pull request by Kajal-techy**
Hello
Kindly review my PR.

### CODE REVIEW

File: e2e/src/app.e2e-spec.ts [hunk 1]
```diff
@@ -1,31 +1,79 @@
...
+  /**
+   * Dashboard page E2E testcases
+   */
+
+  it('Carousal should be present', () => {
+    browser.wait(expectedWait.visibilityOf(dashboardPage.getCarousal()),5000);
+   })
+   
+  it('verify product should display', () => {
+    expect(dashboardPage.getPriceLable().getText()).toEqual("Price:");
```
Comment (AakashIsTheLimit) [inline e2e/src/app.e2e-spec.ts:51]: label

File: e2e/src/app.e2e-spec.ts [hunk 2]
```diff
@@ -1,31 +1,79 @@
...
+    browser.ignoreSynchronization = true;
+    expect(browser.getCurrentUrl()).toContain('/dashboard');
+  })
+
+  /**
+   * Dashboard page E2E testcases
+   */
+
+  it('Carousal should be present', () => {
+    browser.wait(expectedWait.visibilityOf(dashboardPage.getCarousal()),5000);
```
Comment (AakashIsTheLimit) [inline e2e/src/app.e2e-spec.ts:47]: space after `,`

File: src/app/product/product.component.html [hunk 3]
```diff
@@ -0,0 +1,5 @@
+<div class="containers_row">
+  <mat-progress-spinner *ngIf="!product" mode="indeterminate"></mat-progress-spinner>
```
Comment (AakashIsTheLimit) [inline src/app/product/product.component.html:2]: is there need of mode ?

File: src/app/seller-info/seller-info.component.html [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<mat-card class="mat_card">
```
Comment (AakashIsTheLimit) [inline src/app/seller-info/seller-info.component.html:1]: mat_card class is not defined anywhere

File: src/app/seller-info/seller-info.component.css [hunk 5]
```diff
@@ -0,0 +1,23 @@
+.seller_details {
+    font-size: 180%;
```
Comment (AakashIsTheLimit) [inline src/app/seller-info/seller-info.component.css:2]: font-size should be in rem... and below as well

File: src/app/service/product/product.service.ts [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+export class ProductService {
+
+  url = "http://localhost:8095/api/products";
+  sessionInfo: SessionInfo;
+
+  constructor(private httpClient: HttpClient, private sessionService: SessionService) { }
+
+  public getAllProducts(): Observable<any> {
+    let header = new HttpHeaders().set(
+      "Authorization",
```
Comment (AakashIsTheLimit) [inline src/app/service/product/product.service.ts:20]: single quotes and same line

File: src/app/product/product.component.html (hunk 3 above)
Comment (Kajal-techy) [reply src/app/product/product.component.html:2]: There are tow types of mode determinate and indeterminate.
determinate works as per progress but indeterminate will work until timeout wouldn't happen.

**Pull request merged**